# contains several solutions of the same tcp positions
sensor_msgs/JointState[] configuration
